odoo/api.py

```diff
@@ -632,24 +632,44 @@ def __call__(self, cr=None, user=None, context=None, su=None):
         return Environment(cr, uid, context, su, self.uid_origin)
 
     def ref(self, xml_id, raise_if_not_found=True):
-        """ Return the record corresponding to the given ``xml_id``.
+        """ Return the record corresponding to the given ``xml_id``."""
+        return next(iter(self.refs([xml_id], raise_if_not_found=raise_if_not_found).values()), None)
 
-        :param str xml_id: record xml_id, under the format ``<module.id>``
+    def refs(self, xml_ids, model=None, raise_if_not_found=True):
```

Comment:
just an idea.
in my PR for jsonb company dependent, I move the on delete cascade logic for many2one comapny dependent field from the database to ORM. As a result, for models like `account.account` which is used as the comodel for many2one company dependent field, or `res.company` which was never deleted by cascade, we even don't need to check their existance.

Reply:
We are not checking existence today, we just browse. It will fail later if you use an inexistent record .

Reply:
https://github.com/odoo/odoo/blob/4ca79b1549eec988a31b80aa0e9f03e6420e84df/odoo/api.py#L648